### ISSUE TITLE
SwallowSalon & AmatuerAllure support

### ISSUE THREAD

**Issue by chempke** [PAhelper/PhoenixAdult.bundle#110]
I'm having trouble getting this update to work. Matching with AmatuerAllure is random for me. I have tried with Titels and Actors but only a few have worked that way. Unfortunately, SwallowSalon does not work right now and gives errors.

**Comment by PAhelper** [comment]
Yeah I posted it before thorough testing. I think I have it fixed, and will post the fixes soon.

**Comment by PAhelper** [comment]
These should be fixed in the latest code...

**Comment by chempke** [comment]
Yep, it's better but still not perfect. SwallowSalon Actors are not listed but packed in the summary and I still don't figure out how to match this scene for AmatureAllure: 
`Alli Rae POV, Sexy Blonde Beauty Loves a Good Suck & Fuck Session` 
I will post the logfile later..

**Comment by PAhelper** [comment]
Added actor support for Swallow Salon in 65df3449925552555eb3083270e1f9f4b2a36e97. I swear I added that before (because this is mostly a copy/paste of JulesJordan code), but it must have been overwritten at some point.

Amateur Allure does not keep Actors in a format that's easy to pull. If the title is consistent ("Amateur Allure Presents [...] in" I might be able to use that. For now though, AmateurAllure does not have actor support.

As for matching Amateur Allure, it should be the easiest ever. "Amateur Allure - Allie Rae" should get it certainly. It's a shame, these 2 sites have the most advanced search function ever. Right now I'm using the "All words must be included" search, but I might switch to the "Any of these words" search. That will make it more like Gamma Entertainment searches, but the only reason it's a problem with Gamma sites is because they have SOOOOO much content. The specific thing you're looking for gets lost. We probably won't have the same problem here.

**Comment by PAhelper** [comment]
I looked into this, and I can get a lot of the newer actress names from the title because they use a few standard titles:
Actress Names Returns...
Amateur Allure Presents Actress Name...
Amateur Allure Welcomes Actress Name...
Amateur Allure Introduces Actress Name...

But I soon realized many of the older scenes don't follow this at all. As I started trying to add those, it just seemed like an endless rabbit hole. I think it best to just leave actor support off Amateur Allure until a better way is found or they update the site to include them somewhere.
